lib/modules/hosts/soundcloud.js

```diff
@@ -8,21 +7,11 @@ export default {
 	logo: 'https://a-v2.sndcdn.com/assets/images/sc-icons/favicon-2cadd14b.ico',
 	detect: () => true,
 	async handleLink(href) {
-		const { html } = await ajax({
-			url: 'https://soundcloud.com/oembed',
-			data: {
-				url: href,
-				format: 'json',
-				iframe: 'true',
-			},
-			type: 'json',
-		});
-
-		// Get src from iframe html returned
-		const src = $(html).attr('src');
+		const src = string.encode`https://w.soundcloud.com/player/?url=${href}`;
 		return {
 			type: 'IFRAME',
 			embed: src,
+			height: '124px',
```

Comment:
Yeah, was debating between the two, thought 166px looks better for posts, but worse for comments... but sure...